docs/man/rpm-man-template.scd

```diff
@@ -0,0 +1,74 @@
+RPMCOMMAND(1)
+
+# NAME
+rpmcmd - Example rpm man page
+
+# SYNOPSIS
+*rpmcmd* [options] [_ARGUMENT_] ...
+
+# DESCRIPTION
+Describe what *rpmcmd* command does in detail.
+
+Put distinct operational modes to their own subsections.
+
+Typographic conventions:
+- Use bold for literal values such as command name and options:
+*cmd* does X, with option *--do-this* does Y instead
+- Use underline uppercase to denote user arguments: _ARGUMENT_
+- Use underline for literal file names: _hello-1.0-1.noarch.rpm_
+- Use alphabetical order whenever multiple things are listed and
+  there's no contextual reason to do otherwise
+- Put the short variant first when listing option aliases
+- Put an empty line before a new section/option/etc
+- Put text immediately below a section/option/etc (ie without an empty line)
+- Put mandatory elements into <>
+- Put optional elements into [] brackets
+- Do not emphasize separators like ",", "=", "[]", "<>", "|" etc
+
+When in doubt, consult *man-pages*(7).
+
+# OPTIONS
+Describe common options and in this section.
+
+Typographic conventions:
+- Put option aliases on separate lines
+
+*-d*
+*--do-this*
+	Do Y instead of default action.
+
+*-o* <_ARG_>,
+*--output*=<_ARG_>
+	Use file _ARG_ for output.
+
+*--compress*=<*gzip*|*zstd*>
+	Use of *gzip* and *zstd* to compress.
```

Comment:
I'd maybe replace "and" with "or" here, otherwise it reads like "use both" :smile: